src/win32/mouse.cpp

```diff
@@ -16,8 +17,8 @@ void MoveAbsolute(int x, int y) {
     // (65535, 65535) is the bottom right corner
     INPUT input = {0};
     input.type = INPUT_MOUSE;
-    input.mi.dx = x;
-    input.mi.dy = y;
+    input.mi.dx = round(x * 65535 / GetDeviceCaps(GetDC(NULL), HORZRES));
```

Comment:
An integer divided by an integer results in an integer. One possible solution is to use `65535.0` instead of `65535`.